dapps/assets/src/components/contacts/AddContact.vue

```diff
@@ -0,0 +1,96 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <form id="contactForm" ref="contactForm" @submit.prevent="addContact">
+    <evan-swipe-panel
+      ref="addContactPanel"
+      alignment="right"
+      type="default"
+      class="light"
+      :showBackdrop="true"
+      :hideCloseButton="true"
+      :title="$t('_assets.contacts.add-contact-title')"
+      @hide="initState"
+    >
+      <p>{{ '_assets.contacts.add-contact-desc' | translate }}</p>
+        <evan-form-control-input
```

Comment:
Wrong indent. Please check also the next lines.
```suggestion
      <evan-form-control-input
```